### PR TITLE
Update troubleshooting for create cluster failure

### PR DESCRIPTION
**Pull request by jessicaochen**
Provide guidance on where to look for common failure cases for kubeadm

### CODE REVIEW

File: docs/test-failures.md [hunk 1]
```diff
@@ -42,3 +42,14 @@ upcoming changes, and maintainers of other projects can subscribe.
...
+
+# Troubleshooting
+## Create Cluster Fails
+There is an issue in provisioning the cluster.
+
+Test failure will look like:
+```error during make -C /workspace/kubernetes-anywhere WAIT_FOR_KUBECONFIG=y deploy: exit status 2```
+
+To debug further, dig into the collected logs
+1. Determine master was able to set up with kubeadm. Logs are under: /artifacts/master-node-name/serial-1.log
```
Comment (roberthbailey) [inline docs/test-failures.md]: Determine if master was able to be set up with kubeadm.

File: docs/test-failures.md [hunk 2]
```diff
@@ -42,3 +42,14 @@ upcoming changes, and maintainers of other projects can subscribe.
...
+2. Determine of nodes were able to set up with kubeadm. Logs are under: /artifacts/node-name/serial-1.log
```
Comment (roberthbailey) [inline docs/test-failures.md]: Determine if nodes were able to be set up with kubeadm.

File: docs/test-failures.md (hunk 1 above)
Comment (jessicaochen) [reply docs/test-failures.md]: Done.

File: docs/test-failures.md (hunk 2 above)
Comment (jessicaochen) [reply docs/test-failures.md]: Done.

File: docs/test-failures.md [hunk 3]
```diff
@@ -42,3 +42,14 @@ upcoming changes, and maintainers of other projects can subscribe.
 
 Without strict versioning, this contract between projects and upstream
 will always be brittle - so regular communication is a must.
+
+# Troubleshooting
+## Create Cluster Fails
+There is an issue in provisioning the cluster.
+
+Test failure will look like:
+```error during make -C /workspace/kubernetes-anywhere WAIT_FOR_KUBECONFIG=y deploy: exit status 2```
```
Comment (xiangpengzhao) [inline docs/test-failures.md]: When previewing, the two lines are actually in one single line. Please check it. (Adding an empty line between them might fix this.)

File: docs/test-failures.md [hunk 4]
```diff
@@ -42,3 +42,14 @@ upcoming changes, and maintainers of other projects can subscribe.
...
+
+To debug further, dig into the collected logs
+1. Determine if master was able to set up with kubeadm. Logs are under: /artifacts/master-node-name/serial-1.log
```
Comment (xiangpengzhao) [inline docs/test-failures.md]: It'd be good to make the log path back-ticked or quoted I guess.

**Comment by jessicaochen** [comment]
Sounds good. 

@timothysc  - I think this repo needs manual merging (as opposed to robo auto-merge). Do you know who can do it?

**Comment by roberthbailey** [comment]
Please send a follow up PR fixing the comments (I didn't merge sooner because I thought they were useful, but merging on @timothysc's suggestion to merge now and refine).

**Pull request merged**